index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: setup and mobile version skeleton</title>
+</head>
+<body>
+  <section class="header">
+  
+      <img src="https://raw.githubusercontent.com/microverseinc/curriculum-html-css/main/portfolio/images/m1_setup/toolbar-mobile.png?token=GHSAT0AAAAAABR4AJN6X4BQE6U47NXD7DIKYR45DAA" alt="the navigation bar">
+
+  </section>
```

Comment:
- [ ] The toolbar section should be properly developed as required in the project requirements using elements such as `<header>, <nav>, <a>` etc. and not just a screenshot of the Figma template... Please develop the section properly using the right element